Add link to the Spectrum service (#353)
mirumee/ariadne: .github/ISSUE_TEMPLATE/issue_or_bug_report.md

---
name: ✏ Issue or bug report

> about: Use for general issues and bug reports related only to the project's codebase.

Reviewer: ```suggestion
about: Use for bug reports or other issues with the codebase.
```
